### PR TITLE
Us04 kat

### PR DESCRIPTION
**Pull request by Katthaleeya**
I definitely have some conflict on this push and request.  Any advice or suggestions would be appreciated!

### CODE REVIEW

File: src/test/java/com/transmuda/pages/VehiclesPage.java [hunk 1]
```diff
@@ -24,4 +25,17 @@ public VehiclesPage(){
...
+    @FindBy(xpath = "(//ul[@class='dropdown-menu dropdown-menu__action-cell launchers-dropdown-menu'])[1]")
+    public WebElement editInfoIcons;
+
+
+
+
+    @FindBy (xpath = "(//ul[@class='dropdown-menu dropdown-menu__action-cell launchers-dropdown-menu'])[1]")
+    public WebElement dropDownIcons;
+
+
```
Comment (marinabilyk) [inline src/test/java/com/transmuda/pages/VehiclesPage.java:37]: Are those 2 web elements supposed to have an identical locator?

File: src/test/resources/features/US06_VerifyEditIcons_Kat.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
+@B34G12-193
+Feature: Default
+
+		
+
+	@B34G12-191
+	Scenario Outline: Verify users can see edit car info icons from the Vehicle page
+		Edit car info icons from Vehicle page
```
Comment (marinabilyk) [inline src/test/resources/features/US06_VerifyEditIcons_Kat.feature:8]: I believe this line is a duplicate of a Scenario Outline title/description

File: src/test/java/com/transmuda/step_definitions/US06_StepDefs.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    public void userSeesEditInfoIcons() {
+
+        String actualIcons = vehiclesPage.dropDownIcons.getAttribute("class");
+
+        String expectedIcons = "dropdown-menu dropdown-menu__action-cell launchers-dropdown-menu";
+
+        Assert.assertEquals(actualIcons,expectedIcons);
+
+    }
+}
```
Comment (marinabilyk) [inline src/test/java/com/transmuda/step_definitions/US06_StepDefs.java:59]: So dots and icons are the same elements with the same locator?

File: src/test/java/com/transmuda/pages/VehiclesPage.java (hunk 1 above)
Comment (Katthaleeya) [reply src/test/java/com/transmuda/pages/VehiclesPage.java:37]: That's true! Let me fix it. Thank you Maryna

**Pull request closed without merging**